mkContainer/mkContainer.nix

```diff
@@ -151,6 +151,10 @@ let
           "${environment}/activate"
           "--env"
           environment
+          "--env-cache"
+          "/tmp"
```

Comment:
I'm sure this was in the container when I inspected it locally but apparently not and we need to create it ourselves: https://github.com/NixOS/nixpkgs/issues/257172

It's also a pain to test locally because the proxy container needs the changes pushed to GitHub 😭

Reply:
Now created in a separate commit.

Reply:
> It's also a pain to test locally because the proxy container needs the changes pushed to GitHub 😭

Maybe we should reconsider allowing a full flakeref to be passed to the container instead of a git ref/rev